plugins/janus_recordplay.c

```diff
@@ -1668,6 +1708,22 @@ static void *janus_recordplay_handler(void *data) {
 					janus_recorder_encrypted(rc);
 				session->vrc = rc;
 			}
+			/* Check if we should record data too */
+			data = (janus_sdp_mline_find(offer, JANUS_SDP_APPLICATION) != NULL);
+			if(data) {
+				textdata = dotextdata ? json_is_true(dotextdata) : TRUE;
+				char filename[256];
+				if(filename_text != NULL) {
+					g_snprintf(filename, 256, "%s-data", filename_text);
+				} else {
+					g_snprintf(filename, 256, "rec-%"SCNu64"-data", id);
+				}
+				rec->drc_file = g_strdup(filename);
```

Comment:
nit: you could simplify this (and get rid of the intermediate `filename` buffer) with just:
```
if(filename_text != NULL) {
   rec->drc_file = g_strdup_printf("%s-data", filename_text);
} else {
  rec->drc_file = g_strdup_printf("rec-%"SCNu64"-data", id);
}
```

Reply:
Oh I wasn't aware of `g_strdup_printf`: since we do the same thing in a few other places as well (core and plugins), for the moment I'll keep this as it is, and then once we merge I'll change them all in a single commit. Thanks for the heads up!